src/sources/file.rs

```diff
@@ -161,6 +165,7 @@ const fn default_lines() -> usize {
 
 #[derive(Debug)]
 pub(crate) struct FinalizerEntry {
+    pub(crate) file_name: String,
```

Comment:
It looks like this maps to a `FileFingerprint`, so it could be good to avoid the "name" naming since it will be more of an opaque identifier.

Reply:
Oh whoops, I think I misread the diff there. That makes sense.